llvm-spirv/test/DebugInfo/X86/live-debug-variables.ll

```diff
@@ -3,6 +3,7 @@
 ; RUN: llvm-spirv -r %t.spv -o - | llvm-dis -o %t.ll
 
 ; RUN: llc -mtriple=x86_64-linux-gnu -filetype=obj -o - %t.ll | llvm-dwarfdump -debug-loc - | FileCheck %s
+; XFAIL: *
```

Comment:
You can cherry-pick change under review: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/650/files